doc/user/content/release-notes.md

```diff
@@ -58,8 +58,11 @@ Wrap your release notes at the 80 character mark.
 - [`TAIL`](/sql/tail) now supports timestamp progress.
 - Log messages are no longer emitted to stderr if an explicit `--log-file` is
   supplied at startup. {{ gh 4777 }}
+- The systemd package configured by our apt package now writes all logs to
+  the systemd journal, instead of a file in the mzdata directory. {{ gh 4781 }}
```

Comment:
Just realized that these aren't valid shortcodes. Mind fixing them up to include the `%`?

Reply:
oh, sure.